core-bundle/src/Routing/Content/PageRoute.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Routing\Content;
+
+use Contao\CoreBundle\ContaoCoreBundle;
+use Contao\PageModel;
+use Symfony\Cmf\Component\Routing\RouteObjectInterface;
+use Symfony\Component\Routing\Route;
+
+class PageRoute extends Route implements RouteObjectInterface
+{
+    /**
+     * @var PageModel
+     */
+    private $page;
+
+    /**
+     * @var string
+     */
+    private $languagePrefix;
+
+    /**
+     * @var string
+     */
+    private $urlSuffix;
+
+    /**
+     * The referenced content object.
+     */
+    private $content;
+
+    public function __construct(PageModel $page, $content = null)
+    {
+        $page->loadDetails();
+
+        $defaults = [
+            '_token_check' => true,
+            '_controller' => 'Contao\FrontendIndex::renderPage',
```

Comment:
I thought that was deprecated now?

Reply:
Not for the default `PageRegular` behavior. It _should_ become deprecated at some point, but I think that's too much for this PR.

Reply:
Okay, resolving conversation then :)